Explorer/Assets/DCL/SDKComponents/CameraModeArea/Tests/CameraModeAreaHandlerSystemShould.cs

```diff
@@ -0,0 +1,163 @@
+using Arch.Core;
+using CRDT;
+using CrdtEcsBridge.Components;
+using DCL.CharacterCamera;
+using DCL.CharacterTriggerArea.Components;
+using DCL.ECSComponents;
+using DCL.SDKComponents.CameraModeArea.Systems;
+using DCL.Utilities;
+using ECS.Prioritization.Components;
+using ECS.TestSuite;
+using NUnit.Framework;
+using Vector3 = Decentraland.Common.Vector3;
+
+namespace DCL.SDKComponents.CameraModeArea.Tests
+{
+    public class CameraModeAreaHandlerSystemShould : UnitySystemTestBase<CameraModeAreaHandlerSystem>
+    {
+        private Entity entity;
+        private Entity cameraEntity;
+        private World globalWorld;
+
+        [SetUp]
+        public void Setup()
+        {
+            globalWorld = World.Create();
+            var globalWorldProxy = new WorldProxy();
+            globalWorldProxy.SetWorld(globalWorld);
+
+            cameraEntity = globalWorld.Create(
+                new CRDTEntity(SpecialEntitiesID.CAMERA_ENTITY),
+                new CameraComponent { Mode = CameraMode.ThirdPerson }
+            );
+
+            var cameraEntityProxy = new EntityProxy();
+            cameraEntityProxy.SetEntity(cameraEntity);
+
+            system = new CameraModeAreaHandlerSystem(world, globalWorldProxy, cameraEntityProxy);
+
+            entity = world.Create(PartitionComponent.TOP_PRIORITY);
+            AddTransformToEntity(entity);
+        }
+
+        [Test]
+        public void SetupCharacterTriggerAreaCorrectly()
+        {
+            var areaSize = new Vector3
+            {
+                X = 2.5f,
+                Y = 3.8f,
+                Z = 8.5f,
+            };
+
+            var component = new PBCameraModeArea
+            {
+                Area = areaSize,
+                IsDirty = true,
+            };
+
+            world.Add(entity, component);
+
+            system.Update(1);
+
+            Assert.IsTrue(world.TryGet(entity, out CharacterTriggerAreaComponent triggerAreaComponent));
+            Assert.AreEqual(new UnityEngine.Vector3(areaSize.X, areaSize.Y, areaSize.Z), triggerAreaComponent.AreaSize);
+        }
+
+        [Test]
+        public void UpdateCharacterTriggerAreaCorrectly()
+        {
+            var areaSize = new Vector3
+            {
+                X = 5.2f,
+                Y = 8.3f,
+                Z = 5.8f,
+            };
+
+            var component = new PBCameraModeArea
```

Comment:
`Decentraland.Common.Vector3` (not Unity's) can't be made a CONST easily, It should be fine to leave those `var areaSize = new Vector3` for now since they are only used in 2 tests (and with different values).